### PR TITLE
Updated edx ansible vars to split out configs needed for xpro

### PR DESCRIPTION
**Pull request by blarghmatey**

### CODE REVIEW

**Comment by shaidar** [comment]
Should `top` pillar file be included in this?

**Comment by blarghmatey** [comment]
Yes it should. Updated.

File: pillar/edx/ansible_vars/cloud_deployment.sls [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+    EDXAPP_LMS_ENV_EXTRA:
+      <<: *common_env_config
+      BULK_EMAIL_DEFAULT_FROM_EMAIL: mitx-support@mit.edu
+      COURSE_ABOUT_VISIBILITY_PERMISSION: "{{ edxapp_course_about_visibility_permission }}"
+      COURSE_CATALOG_VISIBILITY_PERMISSION: "{{ edxapp_course_catalog_visibility_permission }}"
+      ALLOW_ALL_ADVANCED_COMPONENTS: True
+      FEATURES:
+        <<: *common_feature_flags
+p        ALLOW_COURSE_STAFF_GRADE_DOWNLOADS: true
```
Comment (shaidar) [inline pillar/edx/ansible_vars/cloud_deployment.sls]: Typo?

File: pillar/edx/ansible_vars/residential.sls [hunk 2]
```diff
@@ -107,52 +64,6 @@ edx:
...
-    XQUEUE_MYSQL_PORT: {{ MYSQL_PORT }}
-    XQUEUE_MYSQL_USER: __vault__:cache:mysql-{{ environment }}/creds/xqueue-{{ purpose }}>data>username
-    XQUEUE_UPLOAD_BUCKET: mitx-grades-{{ purpose }}-{{ environment }}
-    xqueue_source_repo: {{ purpose_data.versions.xqueue_source_repo }}
-    xqueue_version: {{ purpose_data.versions.xqueue }}
-    ########## END XQUEUE ########################################
-
     ########## START THEMING ########################################
     EDXAPP_ENABLE_COMPREHENSIVE_THEMING: true
     EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO: '{{ purpose_data.versions.theme_source_repo }}'
```
Comment (shaidar) [inline pillar/edx/ansible_vars/residential.sls]: Doesn't look like `purpose_data` is any longer defined

Comment (blarghmatey) [reply pillar/edx/ansible_vars/residential.sls]: It should be...

**Comment by shaidar** [comment]
👍

**Pull request merged**